Packages/manifest.json

```diff
@@ -13,6 +13,7 @@
     "com.unity.modules.screencapture": "1.0.0",
     "com.unity.modules.terrain": "exclude",
     "com.unity.modules.terrainphysics": "exclude",
+    "com.unity.modules.ui": "1.0.0",
```

Comment:
Do we need this?

Reply:
yes - the canvas scene is broken without it.